package.json

```diff
@@ -60,11 +60,11 @@
     "err-code": "^1.1.2",
     "hamt-sharding": "~0.0.2",
     "ipfs-unixfs": "~0.1.16",
-    "ipld-dag-pb": "~0.17.2",
+    "ipld-dag-pb": "^0.18.0",
     "multicodec": "~0.5.1",
     "multihashing-async": "~0.7.0",
-    "superstruct": "~0.6.1",
-    "rabin-wasm": "~0.0.4"
+    "rabin-wasm": "0.0.8",
```

Comment:
Any reason why this is not prefixed with `~`?

Reply:
The reason was an `npm install rabin-wasm@latest`. So it should really be `~0.0.8`.